TotalMayhemGames/EscapeRoom
# Added fading for dungeon map

How to test:

- Place the chandelier prefab to the scene together with a switch
- When the chandelier lever is used the map beneath it should light up.
- The revealed areas on the map fade when the chandelier moves away.
